src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/OpMatrix_Unary_Decimal.txt

```diff
@@ -57,13 +57,13 @@ Error({Kind:ErrorKind.InvalidArgument})
 2.000000000000000000000002
 
 >> -Date(2000,1,1)
-#SKIP: waiting on https://github.com/microsoft/Power-Fx/issues/1294, expected: 36526
```

Comment:
I understand these results.  Can we also add some tests for the numeric values, just to make sure that is coming out correctly.  In this case, `Value(-Date(2000,1,1) )` returning -36526, and the others.

Reply:
Added tests with the value and more comments